addons/arsenal/functions/fnc_addRightPanelButton.sqf

```diff
@@ -42,14 +42,14 @@ if (_position >= 0 && _position <= 9) then {
     private _cfgWeapons = configFile >> "CfgWeapons";
     _items = _items select {
         private _configItemInfo = _cfgWeapons >> _x >> "ItemInfo";
-           
+
         _x isKindOf ["CBA_MiscItem", _cfgWeapons] && {getNumber (_configItemInfo >> "type") in [TYPE_MUZZLE, TYPE_OPTICS, TYPE_FLASHLIGHT, TYPE_BIPOD]} ||
         {getNumber (_configItemInfo >> "type") in [TYPE_FIRST_AID_KIT, TYPE_MEDIKIT, TYPE_TOOLKIT]} ||
         {getText (_cfgWeapons >> _x >> "simulation") == "ItemMineDetector"}
     };
-    
+
     _return = _position;
-    GVAR(customRightPanelButtons) set [_position, [_items apply {toLower _x}, _picture, _tooltip]];
+    GVAR(customRightPanelButtons) set [_position, [_items, _picture, _tooltip]];
```

Comment:
This is a API function.
You cannot really rely on the input classnames being correct case.
You should convert them to config-case